### ISSUE TITLE
SASL support to collectd-virt plugin

### ISSUE THREAD

**Issue by bronhaim** [collectd/collectd#1926]
*   Version of collectd:
collectd-5.5.2-1

*   Operating system / distribution:
f24

## Expected behavior
If libvirt is configured with `auth_unix_rw="sasl"` the connect logic needs to
use auth parameters. This logic is not part of current collectd-virt plugin.
Therefore, the plugin does not do anything if sasl is set.

## Actual behavior
Having the option to configure user and password and use it to communicate with virsh.

## Steps to reproduce
 Basically vdsm package configures libvirt to use sasl - so easily install vdsm rpm, running vdsm-tool configure and then try to use collectd-virt plugin will reproduce the issue.

**Comment by bronhaim** [comment]
Apparently it works when I remove the "Connection "qemu:///system"" parameter from virt conf. So the RO connection works fine. There is no need to support the authentication part only for reading stats.
